pkg/ipamd/ipamd.go

```diff
@@ -344,7 +342,7 @@ func (c *IPAMContext) nodeInit() error {
 	vpcCIDRs := c.awsClient.GetVPCIPv4CIDRs()
 
 	for _, cidr := range vpcCIDRs {
```

Comment:
I think this `for` loop can be removed, remove `pbVPCcidrs`, and just use `vpcCIDRs` throughout the rest of this function.